modules/auxiliary/scanner/http/jboss_vulnscan.rb

```diff
@@ -103,6 +105,9 @@ def check_app(app)
       when res.code == 301, res.code == 302
         print_status("#{rhost}:#{rport} #{app} is redirected (#{res.code}) to #{res.headers['Location']} (not following)")
       else
+        if res.code == 500 && app == "/invoker/readonly"
```

Comment:
This might be better as another `when` branch, rather than increasing complexity of the `else` branch.

```ruby
      when (res.code == 500 && app == '/invoker/readonly')
        print_good("#{rhost}:#{rport} #{app} responded (#{res.code})")
      else
         print_status("#{rhost}:#{rport} Don't know how to handle response code #{res.code}")
      end
```